google/cloud/testing_util/CMakeLists.txt

```diff
@@ -37,6 +37,8 @@ add_library(
     mock_backoff_policy.h
     mock_fake_clock.cc
     mock_fake_clock.h
+    opentelemetry.cc
```

Comment:
maybe `opentelemetry_matchers.{h,cc}` that name is bound to conflict with something else?

Reply:
Updated.